server/athenian/api/precompute/prometheus.py

```diff
@@ -0,0 +1,45 @@
+"""Prometheus support module for precompute batch commands."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from prometheus_client import CollectorRegistry, Histogram, push_to_gateway
+
+
+def get_metrics() -> Metrics:
+    """Get the singleton collection of metrics used by precomputer batch jobs."""
+    global _metrics
+    return Metrics(
+        precompute_account_seconds=Histogram(
+            "precompute_account_seconds",
+            "Duration of the precompute operation on a single account.",
+            ["account", "github_account", "fresh_account"],
+            registry=_get_registry(),
+            buckets=[1, 2, 3, 5, 10, 15, 20, 30, 60, 120, 180, 300, 600, 1200],
+        ),
+    )
+
+
+def push_metrics(gateway: str, job: str) -> None:
+    """Push current metrics to Prometheus Pushgateway."""
+    push_to_gateway(gateway, job, _get_registry())
```

Comment:
```suggestion
    push_to_gateway(gateway, job, REGISTRY)
```
from `prometheus_client.registry` => we can remove our own.

Reply:
here https://github.com/prometheus/client_python#exporting-to-a-pushgateway doc says to not use default registry with pushgateway since that includes host metrics which are meant for a long running process

Reply:
OK, then let's just remove the singleton function.

Reply:
done